### PR TITLE
Career folder updated

### PR DESCRIPTION
**Pull request by SaranshDawra**
Fixed: #1205 
@vegetabill Please review my Pull Request.

### CODE REVIEW

File: career/linked-in.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Example: 60 minutes
 
 Here are links to lessons that should be completed before this lesson:
 
-- [Resume writing](https://github.com/Techtonica/curriculum/blob/master/resume-writing/resume-writing.md)
+- [Resume writing](https://github.com/Techtonica/curriculum/blob/master/career/resume-writing.md)
```
Comment (vegetabill) [inline career/linked-in.md]: please turn this into a relative link

Comment (Kundan28) [reply career/linked-in.md]: I think that could be mentioned in the issue from next time.

**Comment by SaranshDawra** [comment]
@vegetabill I have made the requested changes. Please review it.

File: career/linked-in.md (hunk 1 above)
Comment (vegetabill) [reply career/linked-in.md]: @Kundan28 that would not be realistic for me to know the link format of every document. I have linked in our [style guide](https://github.com/Techtonica/curriculum/blob/master/_templates/instructions.md#links) in the instructions so it is more clear.

**Comment by vegetabill** [comment]
@SaranshDawra was there a reason you force-pushed? I am not seeing new changes and there are now conflicts.

**Comment by SaranshDawra** [comment]
@vegetabill There were conflicts before forcing the push. I was trying to combine all the commits. Can you guide me through this?

File: career/linked-in.md (hunk 1 above)
Comment (Kundan28) [reply career/linked-in.md]: Oh, okay. @vegetabill

**Comment by Kundan28** [comment]
@SaranshDawra Please resolve the conflicts and push your latest commits again as we don't see them.

**Comment by Kundan28** [comment]
@SaranshDawra This is not a good practice.
Please open it and work on it rather than creating a new one for same stuff.

**Pull request closed without merging**